custom_components/google_home/const.py

```diff
@@ -23,6 +28,12 @@
 SENSOR = "sensor"
 PLATFORMS = [SENSOR]
 
+# Services
+SERVICE_DELETE_ALARM = "delete_alarm"
+SERVICE_DELETE_TIMER = "delete_timer"
+SERVICE_ATTR_ID_ALARM = "alarm_id"
+SERVICE_ATTR_ID_TIMER = "timer_id"
```

Comment:
I'm pretty sure I added this before.
```suggestion
SERVICE_ATTR_ALARM_ID = "alarm_id"
SERVICE_ATTR_TIMER_ID = "timer_id"
```